Monika After Story/game/script-stories.rpy

```diff
@@ -126,36 +151,76 @@ label mas_story_begin:
     return
 
 label mas_story_unlock_random:
+   call mas_story_unlock_random_cat()
+   return
+
+label mas_scary_story_unlock_random:
+   call mas_story_unlock_random_cat(scary=True)

```

Comment:
could you add a dev function that can unlock and see all the stories?

calling this label works but it gives me them randomly, which is a minor annoyance when trying to test/review some of the action code.

Reply:
sure it'll be used to unlock any locked story.